src/Makefile.qt.include

```diff
@@ -161,6 +160,9 @@ QT_MOC_CPP = \
   qt/moc_walletmodel.cpp \
   qt/moc_walletview.cpp
 
+QT_MOC_CPP_BIP70 = \
```

Comment:
IMO would be cleaner to just `QT_MOC_CPP +=` (and `BITCOIN_QT_CPP +=`) the actual files below, keeping all the stuff together.

Reply:
I agree, but I just followed the flow already used in the makefile for the wallet, to not have to move large blocks around. A refactor like that could be done separately.